src/main/java/edu/ie3/datamodel/io/source/influxdb/InfluxDbWeatherSource.java

```diff
@@ -152,24 +156,25 @@ public InfluxDbWeatherSource(
     return measurementsMap.get(MEASUREMENT_NAME_WEATHER).stream()
         .map(
             fieldToValue -> {
-              Optional<Point> coordinate =
-                  coordinateSource.getCoordinate(
-                      Integer.parseInt(fieldToValue.remove(coordinateIdColumnName)));
-              if (!coordinate.isPresent()) return null;
-              fieldToValue.putIfAbsent("uuid", UUID.randomUUID().toString());
-
-              /* The factory expects camel case id's for fields -> Convert the keys */
-              Map<String, String> camelCaseFields =
+              /* The factory expects flat case id's for fields -> Convert the keys */
+              Map<String, String> flatCaseFields =
                   fieldToValue.entrySet().stream()
                       .collect(
                           Collectors.toMap(
-                              entry -> StringUtils.snakeCaseToCamelCase(entry.getKey()),
+                              entry ->
+                                  StringUtils.snakeCaseToCamelCase(entry.getKey()).toLowerCase(),
                               Map.Entry::getValue));
```

Comment:
Do you think it might make sense to extract this to a different function? Would reduce complexity of this function.

Reply:
With the new functionalities in PSU, I can imagine a universal converion tool for naming formats: Convert from an arbitrary format to another using `NamingConvention`. I created a new issue here: https://github.com/ie3-institute/PowerSystemUtils/issues/235

I'd suggest to solve issues like this using this new functionality (in the future), which would also make parts of `StringUtils` obsolete.